### PR TITLE
feat: support gleam

### PR DESCRIPTION
**Pull request by wesleimp**
Language server for Gleam Programming Language

https://github.com/gleam-lang/gleam

cc: @lpil

### CODE REVIEW

**Review by lpil — APPROVED** [review]
✨

**Comment by justinmk** [comment]
CI failures need to be fixed

File: lua/lspconfig/server_configurations/gleam.lua [hunk 1]
```diff
@@ -0,0 +1,22 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = {'gleam', 'lsp'},
```
Comment (glepnir) [inline lua/lspconfig/server_configurations/gleam.lua:5]: there should be `cmd = { 'gleam', 'lsp' }`

**Comment by glepnir** [comment]
closed as #2266

**Pull request closed without merging**